web-common/pom.xml

```diff
@@ -106,6 +106,16 @@
                                     <sourceFile>${project.build.outputDirectory}/META-INF/resources/${localeDir}favicon.ico</sourceFile>
                                     <destinationFile>${project.build.outputDirectory}/META-INF/resources/${localeDir}favicon.${projectBuildNumber}.ico</destinationFile>
                                 </fileSet>
+                                
+                                <!-- Rename Confidential image -->
+                                <fileSet>
```

Comment:
I thought you changed to test meaning you should remove the images

Reply:
Done